vars/provisionNodes.groovy

```diff
@@ -113,14 +113,22 @@ def call(Map config = [:]) {
                               chmod 600 /localhome/jenkins/.ssh/{authorized_keys,id_rsa*}
                               chown -R jenkins.jenkins /localhome/jenkins/.ssh
                               echo \\"jenkins ALL=(ALL) NOPASSWD: ALL\\" > /etc/sudoers.d/jenkins
+                              yum -y install yum-utils
+                              pkgs=\\"openpa libfabric mercury pmix ompi\\"
+                              for ext in \\$pkgs; do
+                                  rm -f /etc/yum.repos.d/build.hpdd.intel.com_job_daos-stack_job_\\"\\${ext}\\"_job_*.repo
+                                  yum-config-manager --add-repo=http://build.hpdd.intel.com/job/daos-stack/job/\\"\\${ext}\\"/job/master/lastSuccessfulBuild/artifact/artifacts/
+                                  echo \\"gpgcheck = False\\" >> /etc/yum.repos.d/build.hpdd.intel.com_job_daos-stack_job_\\"\\${ext}\\"_job_master_lastSuccessfulBuild_artifact_artifacts_.repo
```

Comment:
Should not hard code in path here.  This needs to work with build-stage/build-dev, etc.